protector/proxy/request_handler.py

```diff
@@ -145,6 +152,21 @@ def do_POST(self):
             self.tsdb_query = OpenTSDBQuery(post_data)
             self.headers['X-Protector'] = self.tsdb_query.get_id()
 
+            # Increment metrics based on query start time
+            start_time = dt.datetime.fromtimestamp(self.tsdb_query.get_start_timestamp())
+            now_time = dt.datetime.fromtimestamp(int(round(time.time())))
+            delta_time = now_time - start_time
+            
+            if delta_time.days >= 0:
+                if delta_time.days < 1:
```

Comment:
how about something like this? 
self.protector.TSDB_REQUEST_INTERVAL_DAYS.labels(...).observe(delta_time.days)

Reply:
Good ideea, i have pushed the changes.